### ISSUE TITLE
NVDA fails to echo characters upon pressing Left and Right arrows if Speech history is active

### ISSUE THREAD

**Issue by amirsol81** [jscholes/nvda-speech-history#8]
If the Speech history add-on is active, NVDA fails to echo each character as we press Left/Right arrows. This affects the functionality of Left/Right wherever they are pressed.
I'm running NVDA alpha-19811,ced1faff, but this started a long while ago -- after the release of initial alphas when 2019.3 was made available. I can't tell if it affects 2019.3 as I don't have it, but it does affect all alphas released over the past few months.
I'm running Speech history version 2019.03.30 which, according to its About dialog, has also been tested with NVDA 2019.3.
This is a Windows 10 Version 2004 (OS Build 19041.153), 64-Bit machine, but the issue was present in earlier releases of Windows 10.
I've tried disabling all add-ons and even installed NVDA afresh with Speech history, but the issue persists.

**Comment by jscholes** [comment]
I'm sorry that you're having problems with the add-on.  However, the version you're running was not released by me, and is not officially supported for this exact reason.  Please await the official release supporting NVDA 2019.3 and later.
